### DISCUSSION TITLE
Aqara H1 can't set fast mode

### DISCUSSION THREAD

**Discussion by dml51** [Koenkk/zigbee2mqtt#11688] [Devices]
Device:
Aqara wireless remote switch H1 (double rocker)Aqara wireless remote switch H1 (double rocker)
Xiaomi WRS-R02 Xiaomi WXKG15LM
Sonoff zigbee dongle plus on HA rpi4

Problem:
I don't need the double tab or hold functions on this switch so i'd like to activate fast mode There's a toggle in the ui but it doesn't seem to set anything. I've tried tapping 5 times to wake up the device before setting fast mode, still doesn't work.
Any workaround or should i learn living with the .5s delay?

Thanks!

**Comment by dml51** [comment]
FYI Tapping 5 times left button before setting to fast just works now! Don't know what's changed :x 
Will keep posted if the settings reverts itself.

**Reply by antst** [reply]
I tried everything. Z2M never complains about not delivering a message. Status request returns fast mode, but in reality it is still slow with about 1sec delay.
At the same time, first message from switch comes instantly, once you click, but this is not action.

When you tap 5 times, do you get flashing led?

**Reply by Eischer** [reply]
Wow. thank you a lot. That works for me with the Aqara E1 Switch perfectly. After clicking 5 Times the mode changed from multi to fast automatically
